app/mapa/mapa-client.tsx

```
@@ -0,0 +1,378 @@
+ * - Incluir una barra lateral con la lista de reportes
+ * - Manejar estados de UI (sidebar, etc)
+ */
+
+"use client"
+
+import { useState, useEffect, useCallback } from "react"
+import { Button } from "@/components/ui/button"
+import { Card, CardContent } from "@/components/ui/card"
+import { Badge } from "@/components/ui/badge"
+import { Alert, AlertDescription, AlertTitle } from "@/components/ui/alert"
+import { List, Layers, Map, X, ChevronRight, MapPin } from "lucide-react"
```

Biome (2.1.2)

[error] Do not shadow the global "Map" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)